tests/Fixer/Phpdoc/PhpdocTagNoNamedArgumentsFixerTest.php

```diff
@@ -0,0 +1,309 @@
+                /**
+                 * @no-named-arguments New description
+                 */
+                class Foo {}
+                PHP,
+            <<<'PHP'
+                <?php
+                /**
+                 * @no-named-arguments
+                 */
+                class Foo {}
+                PHP,
+            ['description' => 'New description'],
+        ];
+
+        yield 'change description' => [
+            <<<'PHP'
+                <?php
+                /**
+                 * @no-named-arguments New description
+                 */
+                class Foo {}
+                PHP,
+            <<<'PHP'
+                <?php
+                /**
+                 * @no-named-arguments Old description
+                 */
+                class Foo {}
+                PHP,
+            ['description' => 'New description'],
+        ];
+
+        yield 'remove description' => [
+            <<<'PHP'
+                <?php
+                /**
+                 * @no-named-arguments
+                 */
+                class Foo {}
+                PHP,
+            <<<'PHP'
+                <?php
+                /**
+                 * @no-named-arguments Description to remove
+                 */
+                class Foo {}
+                PHP,
+        ];
+
+        yield 'multiple classes' => [
+            <<<'PHP'
+                <?php
+
+                /**
+                 * @no-named-arguments
+                 */
+                class Foo {}
+
+                new class {};
+
+                /**
+                 * @no-named-arguments
+                 */
+                class Bar {}
+                PHP,
+            <<<'PHP'
+                <?php
+                class Foo {}
+
+                new class {};
+
+                class Bar {}
+                PHP,
+        ];
+
+        yield 'tabs and Windows Line endings' => [
+            str_replace(
+                ['    ', "\n"],
+                ["\t", "\r\n"],
+                <<<'PHP'
+                    <?php
+
+                    /**
+                     * @no-named-arguments
+                     */
+                    class Foo {}
+                    PHP,
+            ),
+            str_replace(
+                ['    ', "\n"],
+                ["\t", "\r\n"],
+                <<<'PHP'
+                    <?php
+
+                    class Foo {}
+                    PHP,
+            ),
+            [],
+            new WhitespacesFixerConfig("\t", "\r\n"),
+        ];
+    }
+
+    /**
+     * @dataProvider provideFix80Cases
+     *
+     * @requires PHP >= 8.0
+     */
+    public function testFix80(string $expected, ?string $input = null): void
+    {
+        $this->doTest($expected, $input);
+    }
+
+    /**
+     * @return iterable<string, array{0: string, 1?: string}>
+     */
+    public static function provideFix80Cases(): iterable
+    {
+        yield 'do not add for attribute class' => [
```

Comment:
why not?

Reply:
Please suggest the change, I didn't come up with anything good.

Reply:
my suggestion is to add the annotation to attributes as well.
i do not understand the reasoning why not to do this.

> the whole point of attribute-classes is, that they can be used with named parameters on the usage-side.
> ([ref](https://github.com/kubawerlos/php-cs-fixer-custom-fixers/issues/1058))

doesn't justify why, just claiming so. yet, i fail to find any built-in attribute that matters about argument names. Even the example from above, it literally has only one attribute argument so it doesn't make sense to call it by name. I checked few projects like PHPUnit and i fail to see usage of named arguments for attributes.

Reply:
and yes, we want to be able to add `@no-named-arguments` to attributes too

https://github.com/sebastianbergmann/phpunit/blob/558dfd63e8e823a26453011e50caaef0f6fa8380/src/Framework/Attributes/CoversMethod.php#L17

Reply:
I think it is very common to use named arguments for attributes. Symfony is using them everywhere in attributes, e.g., in command attributes: https://symfony.com/doc/current/console.html#creating-a-command.

So we also use them here in the repo for the `AsCommand` attributes.

Reply:
So in my opinion it is a good default to exclude attributes. But it could be configurable.